pkgs/top-level/emacs-packages.nix

```diff
@@ -1843,5 +1834,6 @@ in
     // melpaPackages self
     // elpaPackages self
     // melpaStablePackages self
+    // orgPackages self
```

Comment:
@mdorman This has `orgPackages` overriding `melpaStablePackages`. Is that what you intended?